src/main/java/com/exactpro/th2/common/schema/factory/AbstractCommonFactory.java

```diff
@@ -634,7 +634,7 @@ public EventID getRootEventId() {
                             .description("Root event")
                             .status(Event.Status.PASSED)
                             .type("Microservice")
-                            .toProto(boxConfiguration.getBookName());
+                            .toProto(boxConfiguration.getBookName(), boxConfiguration.getBoxName());
```

Comment:
Why do we use component name as scope?

Reply:
Scope is alternative of session alias. We added it to organize portioning for event streams. In this case each component will publish own events to own scope, also intermediate components will use scope from message.metadata.parentEventId